vpr/src/device/rr_graph_builder.h

```diff
@@ -239,9 +274,24 @@ class RRGraphBuilder {
      * but simply passed through the flow so that it can be used by downstream tools.
      * The main (perhaps only) current use of this metadata is the fasm tool of symbiflow,
      * which needs extra metadata on which programming bits control which switch in order to produce a bitstream.*/
-
-    MetadataStorage<int>& rr_node_metadata_;
-    MetadataStorage<std::tuple<int, int, short>>& rr_edge_metadata_;
+    /**
```

Comment:
we should add a warning to the comments, in order to avoid any developers to merge the metadate into other internal data:

```
/** .. warning:: The Metadata should stay as an independent data structure than rest of the internal data, e.g., node_lookup! */
```